docs/getting_started/quickstart.md

```diff
@@ -20,9 +20,19 @@ export PROMPTGUARD_API_KEY="..."
 PromptGuard offers two main functions: `sanitize()` and `desanitize()`. `sanitize()` takes a string and returns a sanitized (i.e. encrypted and redacted) version of it, while `desanitize()` takes a sanitized string and returns the original string.
 
 ```python
+import promptguard as pg
+
+input_text = "Some string with sensitive PII"
+sanitized_response = pg.sanitize(text = input_text)
 ```
+The `sanitized_response` contains both the `sanitized_text` as well as `secure_context` which must be passed to the followup call to `desanitize()`. 
```

Comment:
Please add an example of the `sanitized_response` here, and describe what its contents are

Maybe a new code block with

```
> print sanitized_response
{contents_of_response}
```

Then please go through the contents and describe what they are